core/utils/ethabi_test.go

```diff
@@ -568,7 +571,9 @@ func TestParseDecimalString(t *testing.T) {
 		{"12072e-4", "1"},
 		{"1.2072e+20", "120720000000000000000"},
 		{"-1.2072e+20", "-120720000000000000000"},
-		{"1.55555555555555555555e+20", "155555555555555540992"},
+		{"1.55555555555555555555e+20", "155555555555555555555"},
```

Comment:
This is a little odd and what I noticed in the first place: the loss of precision was being tested as expected behavior. :confused: